darts/models/forecasting/baselines.py

```diff
@@ -236,13 +250,27 @@ def fit(
             past_covariates=past_covariates,
             future_covariates=future_covariates,
         )
-        for model in self.models:
-            kwargs = dict(series=series)
-            if model.supports_past_covariates:
-                kwargs["past_covariates"] = past_covariates
-            if model.supports_future_covariates:
-                kwargs["future_covariates"] = future_covariates
-            model.fit(**kwargs)
+        if self.retrain_forecasting_models:
+            # Some models may need to be 'reset' to allow being retrained from scratch, especially torch-based models
+            self.forecasting_models: List[ForecastingModel] = [
```

Comment:
This line is actually unnecessary since the constructor raise an error if retrain_forecasting_models=True with some "already trained" model. I removed it

Reply:
👍